src/js/view/controls/controls.js

```diff
@@ -316,6 +337,11 @@ export default class Controls {
                 case 70: // f-key
                     api.setFullscreen();
                     break;
+                case 191: // ? key
+                    if (!OS.mobile) {
```

Comment:
there are no keys in mobile. I think we should check for existence `if (this.shortcutsTooltip) {`
this also covers the mobile case, since shortcutsTooltip is only instantiated when `!OS.mobile`

Reply:
Agreed, that would be more sustainable.